javascript/javascript1/week3/activities.js

```diff
@@ -0,0 +1,51 @@
+const activities = [];
+function addActivity(date , activity , duration){
+activities.push( 
+    {
+        date : date,
+        activity : activity,
+        duration : duration,
+    }
+);
+}
+console.log(activities);
+
+//calling function for adding activities in Array
+addActivity("23/7-2021" ,"Youtube " , 30)
+addActivity("25/7-2021" ,"Facebook " , 15)
+addActivity("2/8-2021" ,"TikTok" , 25)
+addActivity("3/7-2021" ,"whattsapp" , 15)
+addActivity("6/8-2021" ,"Instagram" , 40)
+
+//show Status
+let sumOfActivitiesDuration = 0;
+function showStatus(activities)
+{
+  for(let i = 0 ; i < activities.length ;i++) 
+  {
+      sumOfActivitiesDuration = sumOfActivitiesDuration + activities[i].duration;
```

Comment:
You can use the shorthand `+=` here:
`sumOfActivitiesDuration += activities[i].duration;

Reply:
Applied shorthand.